source/usage-examples/findOne.go.txt

```diff
@@ -0,0 +1 @@
+map[imdb:map[id:368226 rating:3.5 votes:25673] title:The Room
```

Comment:
Does mongo-driver/bson or other package provide a helper method to convert this to JSON which is a generally more human-readable format?

Reply:
Thanks for the note. I agree, converting this to JSON makes sense. 

As [`bson.M`](https://pkg.go.dev/github.com/mongodb/mongo-go-driver@v1.2.1/bson#M) is handled as a `map[string]interface{}` when encoding and decoding, I think the standard library `json.Marshalndent()` is the way to do print it as JSON (couldn't find a simpler way to do this in the `bson` package, but maybe external review will yield a simpler solution).

Fix:

Going forward, I will try and think more about the readability of my code output.